azure/functions/decorators/function_app.py

```diff
@@ -1373,3 +1374,105 @@ def decorator():
             return decorator()
 
         return wrap
+
+    def read_table(self,
+                   arg_name: str,
+                   connection: str,
+                   table_name: str,
+                   row_key: Optional[str] = None,
+                   partition_key: Optional[str] = None,
+                   data_type: Optional[
+                       Union[DataType, str]] = None) -> Callable:
+        """
+        The read_table decorator adds :class:`TableInput` to the
+        :class:`FunctionBuilder` object
+        for building :class:`Function` object used in worker function
+        indexing model. This is equivalent to defining TableInput
+        in the function.json which enables function to read a table in
+        an Azure Storage or Cosmos DB account
+        All optional fields will be given default value by function host when
+        they are parsed by function host.
+
+        Ref:
+        https://docs.microsoft.com/en-us/azure/azure-function/functions-bindings-storage-table
```

Comment:
opening link return 404, also en-us can be removed, language will be determined by dns server
<img width="1039" alt="image" src="https://user-images.githubusercontent.com/12449837/167009903-77ace2c0-d37f-4ed9-aeb2-b59faad8666b.png">

Reply:
added aka.ms link.